packages/core/src/item.ts

```diff
@@ -278,6 +278,32 @@ export interface AuditResult {
     fieldIndex: number;
 }
 
+export type ItemHistoryFieldsChangedOption = "name" | "vaultId" | "fields" | "tags";
+
+export class ItemHistory extends Serializable {
+    constructor(vals: Partial<ItemHistory> = {}) {
+        super();
+        Object.assign(this, vals);
+    }
+
+    date: Date = new Date();
+
+    updatedBy: AccountID = "";
+
+    fieldsChanged: ItemHistoryFieldsChangedOption[] = [];
```

Comment:
I don't think we need to store this here as we can easily do a diff later.

Reply:
We wouldn't be able to tell what changed for the "oldest" change, though, because there's no older reference.

Reply:
Good point! But honestly, only knowing that the fields changed an not which fields and what from isn't really that helpful either. Also, we can work around that problem easily by simply storing 11 history items but only displaying 10.